frontend/src/app/auth/components/login-page/login-page.component.ts

```diff
@@ -77,13 +107,29 @@ export class LoginPageComponent implements OnInit, OnDestroy {
       this.toastrService.error('Invalid values');
       return;
     }
-    this.authService.loginUser(this.userLogin)
-    .pipe(takeUntil(this.unsubscribe$))
-    .subscribe((resp) => {
-      if(resp.ok){
-        const token = resp.body;
-        this.localStorage.setItem(this.localStorageKeys.token, token?.accessToken as string);
-      }
-    });
+
+    this.userLogin = {
+      email: this.loginForm.get('emailControl')?.value,
+      password: this.loginForm.get('passwordControl')?.value,
+    };
+
+    this.authService
+      .loginUser(this.userLogin)
+      .pipe(takeUntil(this.unsubscribe$))
+      .subscribe((resp) => {
+        if (resp.ok) {
+          const token = resp.body;
+          this.localStorage.setItem(
+            this.localStorageKeys.token,
+            token?.accessToken as string,
+          );
+        }
```

Comment:
No localstorage moves in component, move it to authService or another one

Reply:
Done